python/mxnet/ndarray/numpy/_op.py

```diff
@@ -5308,3 +5307,57 @@ def nan_to_num(x, copy=True, nan=0.0, posinf=None, neginf=None, **kwargs):
         return _npi.nan_to_num(x, copy=copy, nan=nan, posinf=posinf, neginf=neginf, out=None)
     else:
         raise TypeError('type {} not supported'.format(str(type(x))))
+
+
+@set_module('mxnet.ndarray.numpy')
+def diagflat(arr, k=0):
+    """
+    Create a two-dimensional array with the flattened input as a diagonal.
+    Parameters: 
+    
+    v : array_like
+    
+    Input data, which is flattened and set as the k-th diagonal of the output.
+    k : int, optional
+    
+    Diagonal to set; 0, the default, corresponds to the "main" diagonal, a positive (negative) k giving the number of the diagonal above (below) the main.
+    
+    Returns:    
```

Comment:
```Python
    Returns
    -------
```

Reply:
For docs you can almost copy from : https://github.com/numpy/numpy/blob/v1.17.0/numpy/lib/twodim_base.py#L288-L345.

Reply:
get

Reply:
> For docs you can almost copy from : https://github.com/numpy/numpy/blob/v1.17.0/numpy/lib/twodim_base.py#L288-L345.

get